packages/create-drupal-decoupled/package.json

```diff
@@ -43,5 +43,9 @@
     "rimraf": "^6.0.1",
     "typescript": "^5.5.3",
     "typescript-eslint": "^7.16.1"
+  },
+  "engines": {
+    "node": ">=20.15.1",
+    "yarn": ">=1.22.5"
```

Comment:
I tried to use pnpm and got the following error:

```
! The local project doesn't define a 'packageManager' field. Corepack will now add one referencing pnpm@9.5.0+sha512.140036830124618d624a2187b50d04289d5a087f326c9edfc0ccd733d76c4f52c3a313d4fc148794a2a9d81553016004e6742e8cf850670268a7387fc220c903.
! For more details about this field, consult the documentation at https://nodejs.org/api/packages.html#packagemanager

```

Reply:
Also I'm being picky here but; is there any way to get rid of these warnings?.

```
yarn install v1.22.22
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
[4/5] 🔗  Linking dependencies...
warning " > typescript-eslint@7.16.1" has incorrect peer dependency "eslint@^8.56.0".
warning "typescript-eslint > @typescript-eslint/eslint-plugin@7.16.1" has incorrect peer dependency "eslint@^8.56.0".
warning "typescript-eslint > @typescript-eslint/parser@7.16.1" has incorrect peer dependency "eslint@^8.56.0".
warning "typescript-eslint > @typescript-eslint/utils@7.16.1" has incorrect peer dependency "eslint@^8.56.0".
warning "typescript-eslint > @typescript-eslint/eslint-plugin > @typescript-eslint/type-utils@7.16.1" has incorrect peer dependency "eslint@^8.56.0".
[5/5] 🔨  Building fresh packages...
✨  Done in 0.70s.
```

Reply:
I don't think so, because it looks like eslint introduced a breaking change about the config format, and even the plugins works It looks like we need to wait to them to update they dependencies list:(

Reply:
Gotcha, and yeah eslint changed its config format with 8.x and is the default on 9.x. 

I saw this though -> https://github.com/typescript-eslint/typescript-eslint/issues/8211 if we downgrade to eslint 8 would these errors go away?

Also just a reminder that this package.json is missing the `packageManager` field for the corepack functionality to work.